### PR TITLE
update registration controller

### PR DESCRIPTION
**Pull request by haoqing0110**

### CODE REVIEW

File: pkg/addonmanager/controllers/registration/controller.go [hunk 1]
```diff
@@ -157,10 +157,17 @@ func (c *addonConfigurationController) sync(ctx context.Context, syncCtx factory
 		managedClusterAddonCopy.Status.Namespace = managedClusterAddonCopy.Spec.InstallNamespace
 	}
 
+	meta.SetStatusCondition(&managedClusterAddonCopy.Status.Conditions, metav1.Condition{
+		Type:    "RegistrationApplied",
+		Status:  metav1.ConditionTrue,
+		Reason:  "RegistrationApplied",
```
Comment (zhiweiyin318) [inline pkg/addonmanager/controllers/registration/controller.go]: use the type and reason defined in api repo

Comment (haoqing0110) [reply pkg/addonmanager/controllers/registration/controller.go]: fixed

**Comment by qiujian16** [comment]
/approve

@zhiweiyin318 for tagging

**Pull request merged**